core/src/consensus.rs

```diff
@@ -73,14 +83,20 @@ pub(crate) struct ComputedBankState {
 }
 
 #[frozen_abi(digest = "2ZUeCLMVQxmHYbeqMH7M97ifVSKoVErGvRHzyxcQRjgU")]
-#[derive(Serialize, AbiExample)]
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, AbiExample)]
 pub struct Tower {
     node_pubkey: Pubkey,
     threshold_depth: usize,
     threshold_size: f64,
```

Comment:
I wonder these should be serialized, assuming these could be dynamic in the future?

Reply:
yeah so if these change in the future, we'll need some sort of rolling upgrade to make sure people all use the same values at the same time...could be tricky